content/en/docs/Writing policies/verify-images/sigstore/_index.md

```diff
@@ -46,6 +46,9 @@ spec:
                 MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAE8nXRh950IZbRj8Ra/N9sbqOPZrfM
                 5/KAQN0/KjHcorm/J5yctVd7iEcnessRQjU917hmKO6JWVGHpDguIyakZA==
                 -----END PUBLIC KEY-----
+              rekor:
+                url: https://rekor.sigstore.dev
+                ignoreTlog: true                
```

Comment:
@vishal-chdhry, tell me again...these values, especially the `ignoreTlog`, is this only for either Cosign 1.x signings or Cosign 2.x if a user has set the flag to not add to the Tlog?

Reply:
@chipzoller, both `ignoreTlog` and ignoreSCT are for Cosign 2.x (they are not present in cosign 1.x). 
By setting `ignoreTlog` to true, you are skipping the check for tlog entry. Tlogs and SCT verification is now done by default in Cosign 2.x so we have to explicitly stop it if we do not want it.

Reply:
But wouldn't this addition be wrong in cases where it was signed with Cosign 2.x and they DID write to the Tlog?

Reply:
@vishal-chdhry ?

Reply:
@chipzoller Sorry, I missed this...
Yes it would be wrong to do so, I think it would be better to have a separate example that describes how we ignore Tlogs and SCR